README.md

```diff
@@ -1,471 +1,47 @@
-* **client.id** : Kafka producer Client ID
-* **security.providers** : 
-* **ssl.enabled.protocols** : SSL Enabled protocols TLSv1.2, TLSv1.3
-* **ssl.endpoint.identification.algorithm** : SSL endpoint Identification algorithm. Leave default value is you don't want to sent it.
-* **ssl.keymanager.algorithm** : The algorithm used by key manager factory for SSL connections.
-* **ssl.protocol** : The SSL protocol used to generate the SSLContext.
-* **ssl.keystore.type** : Type of the repository of security certificates.
-* **ssl.provider** : The name of the security provider used for SSL connections.
-* **timeout.millis** : Max time trying to poll before considering current partitions empty.
-* **max.poll.interval.ms** : The maximum delay between invocations of poll(). If poll() is not called before 
-  expiration of this timeout, then the consumer is considered failed and the group will rebalance. This value should 
-  be less than timeout.millis.
-* **group.id** : Specifies the name of the consumer group the consumers will belong to.
-
-![Kafka Consumer Configuration](/Kafka_Consumer_Properties.png)
-
->KLoadGen Consumer Sampler is compatible with **JSON** and **AVRO** messages. If **JSON** is being used, no extra 
-> configuration is 
-> needed, sampler will work with default deserialization settings. If **AVRO** is being used, some of the following 
-> components must to be configured in order to define how messages will be deserialized.
-
-### Value Schema Deserialization Configuration
-
-This configuration component allows to specify a Name Strategy to get the correct schema from registry and use it to 
-deserialize read messages. It also allows to select a deserializer, if Avro is being used,
-**KafkaAvroDeserializer** must be chosen. (*io.confluent.kafka.serializers.KafkaAvroDeserializer*)
-
-![Value Schema Deserialization Configuration](/Value_Schema_Deserialization_Config.png)
-
-### Value Schema File Deserialization Configuration
-
-This configuration component will allow you to select a schema from a file (.avsc)
-
-AVRO structure will be flattened and shown in the table. You can see 2 columns:
-
-* **Field Name** : Flattened field name compose by all the properties from the root class. Ex: PropClass1.
-PropClass2.PropClass3 *Note* In case to be an array [] will appear at the end.
-* **Field Type** : Field type, like String, Int, Double, Map, Array... **Note** : if the field is an array of basic 
-  types it will be shown as string-array, int-array...
-  
-In this configuration screen you can also choose a deserializer. Make sure to **chose our custom deserializer** when 
-you 
-are using a schema loaded from a file. (*net.coru.kloadgen.serializer.AvroDeserializer*)
+> *¿Tenemos logo de KLoadGen?*
 
-![Value Schema File Deserialization Configuration](/Value_Schema_File_Deserialization_Config.png)
+[Summary](#) · [Getting started](#) · [Usage](#) · [Support](#) · [Special thanks](#) 
```

Comment:
The internal links to each section are missing. This way the menu is there but it doesn't redirect anywhere. It should be this way: "[Summary](#summary)" or if the section has more than a word "[Getting Started](#getting-started)".